neqo-transport/src/packet/mod.rs

```diff
@@ -1225,7 +1240,14 @@ mod tests {
         // The builder adds randomness, which makes expectations hard.
         // So only do a full check when that randomness matches up.
         if retry[0] == sample_retry[0] {
+            #[cfg(not(feature = "fuzzing_t"))]
             assert_eq!(&retry, &sample_retry);
+
+            #[cfg(feature = "fuzzing_t")]
+            assert_eq!(
+                &retry[..retry.len() - 16],
+                &sample_retry[..sample_retry.len() - 16]
+            );
```

Comment:
I'd say that you shouldn't bother with this check.  Just disable the test (or the above assert).